README.md

```diff
@@ -6,19 +6,21 @@
 * [Homepage](http://googlecloudplatform.github.io/google-cloud-php)
 * [API Documentation](http://googlecloudplatform.github.io/google-cloud-php/#/docs)
 
+This client supports the following Google Cloud Platform services at a [General Availability](#versioning) quality level:
```

Comment:
I like this wording, but @omaray should take a look too given the discussion we had offline.